core/src/main/java/org/apache/accumulo/core/file/rfile/RFile.java

```diff
@@ -1193,16 +1192,14 @@ public void close() throws IOException {
 
       if (sampleReaders != null) {
         for (LocalityGroupReader lgr : sampleReaders) {
-          try {
-            lgr.close();
-          } catch (IOException e) {
-            log.warn("Errored out attempting to close LocalityGroupReader.", e);
-          }
+          lgr.close();
         }
       }
 
       try {
         reader.close();
+      } catch (IOException e) {
+        throw new RuntimeException(e);
```

Comment:
Fixed in 5aa4a99a30052c1560d46d3430886d9e5e650d61. Want me to create a ticket to utilize `UncheckedIOException` everywhere in the code for this situation?